app/src/forms/minesattestations/dataService.js

```diff
@@ -62,20 +62,33 @@ const dataService = {
       .throwIfNotFound();
   },
 
-  current: async () => {
-    const form = await Models.Form.query()
-      .first()
-      .withGraphFetched({ metadata: true})
-      .throwIfNotFound();
-
-    const version = await Models.Version.query()
-      .first()
-      .where('formId', form.formId)
-      .withGraphFetched({ statusCodes: true})
-      .modify('orderDescending')
-      .throwIfNotFound();
-
+  current: async (tiny) => {
+    let form;
+    let version;
+
+    if (tiny) {
+      form = await Models.Form.query()
+        .first()
+        .throwIfNotFound();
+      version = await Models.Version.query()
+        .first()
+        .where('formId', form.formId)
+        .modify('orderDescending')
+        .throwIfNotFound();
+    } else {
+      form = await Models.Form.query()
+        .first()
+        .withGraphFetched({ metadata: true})
+        .throwIfNotFound();
+      version = await Models.Version.query()
+        .first()
+        .where('formId', form.formId)
+        .withGraphFetched({ statusCodes: true})
+        .modify('orderDescending')
+        .throwIfNotFound();
+    }
     form.versions = [version];
+    form.formVersionId = version.formVersionId;
```

Comment:
If I remember async/await stuff correctly, it may be more prudent to run those two model queries wrapped inside a Promise.all() instead to allow for potential parallelism. While these queries are honestly very quick as is, may be good practice to keep that structure. That way, you can just easily .then() chain from that to assign version to form, etc, and return.